csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -531,6 +531,7 @@ private void TestMultiThreads()
                 skipModels["tf_nasnet_large"] = "Get preallocated buffer for initializer ConvBnFusion_BN_B_cell_11/beginning_bn/beta:0_331 failed";
                 skipModels["test_zfnet512"] = "System out of memory";
                 skipModels["test_bvlc_reference_caffenet"] = "System out of memory";
+                skipModels["coreml_VGG16_ImageNet"] = "System out of memory";
```

Comment:
Please insert the line below to:
https://github.com/microsoft/onnxruntime/blob/master/csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs#L558
```
   { "coreml_Imputer-LogisticRegression_sklearn_load_breast_cancer", "Can't determine model file name" },
```

It is picked from 3eaec57c3812b2e667f50e77f64a03b3c981e7b0

Reply:
Windows CPU CI Pipeline still failed with the same error after I cherry picked those 4 PRs,
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=168533&view=logs&j=5b022bb4-70a7-5401-8766-a8a7802c7150&t=23e58576-d346-55e7-f729-22ace67bada2&l=98

##[error]Error: The process 'C:\Program Files\dotnet\dotnet.exe' failed with exit code 1
Info: Azure Pipelines hosted agents have been updated to contain .Net Core 3.x (3.0 and 3.1) SDK/Runtime along with 2.1. Unless you have locked down a SDK version for your project(s), 3.x SDK might be picked up which might have breaking behavior as compared to previous versions. 
Some commonly encountered changes are: 
If you're using `Publish` command with -o or --Output argument, you will see that the output folder is now being created at root directory rather than Project File's directory. To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting
##[error]Dotnet command failed with non-zero exit code on the following projects : C:\agent\_work\1\s\csharp\test\Microsoft.ML.OnnxRuntime.Tests\Microsoft.ML.OnnxRuntime.Tests.csproj